luma/core/virtual.py

```diff
@@ -549,3 +549,63 @@ def _flush(self, buf):
                     if byte & 0x01:
                         draw.point((x, y), fill="white")
                     byte >>= 1
+
+
+class character(object):
+    """
+    Abstraction that wraps a device, this class provides a ``text`` property
+    which can be used to set and get a text value allowing the device to be
+    treated as a character style display such as the HD44780 LCD
+
+    If the device is actually a character style device, be careful to provide
+    a font that adheres to the pixel dimensions of the display.
+
```

Comment:
can you add `.. versionadded:: 1.15.0` to the new class

Reply:
I've added the version tag to the three new classes.